spec/TodoList.spec.js

```diff
@@ -1,17 +1,52 @@
-const TodoList = require("../src/TodoList.js")
+const TodoList = require("../src/TodoList.js");
 
 describe("TodoList", () => {
   it("creates a todo item", () => {
     // set up
-    const todoList = new TodoList()
+    const todoList = new TodoList();
     const expected = {
       id: 1,
       text: "turn the heating on!",
-      status: "incomplete"
-    }
+      status: "incomplete",
+    };
     // execute
-    const result = todoList.create("turn the heating on!")
+    const result = todoList.create("turn the heating on!");
     // verify
-    expect(result).toEqual(expected)
-  })
-})
+    expect(result).toEqual(expected);
+  });
+
+  it("list an all to do items", () => {
+    // set up
+    const todoList = new TodoList();
+    todoList.create("turn the heating on!");
+    todoList.create("make the bed");
+
+    const expected = [
+      {
+        id: 1,
+        text: "turn the heating on!",
+        status: "incomplete",
+      },
+      {
+        id: 2,
+        text: "make the bed",
+        status: "incomplete",
+      },
+    ];
+    // execute
+    const result = todoList.allToDoItems();
+    // verify
+    expect(result).toEqual(expected);
+  });
+
+  it("list to do all items for empty todoList", () => {
+    // set up
+    const todoList = new TodoList();
+
+    const expected = [];
+    // execute
+    const result = todoList.allToDoItems();
+    // verify
+    expect(result).toEqual(expected);
+  });
```

Comment:
This is a nice test, I think it could be useful to use `todoList.create` to add a couple of todos to check that `allToDoItems` does return an array with them inside